### PR TITLE
update main.js

### PR DESCRIPTION
**Pull request by kenzoTange**
have some bug

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1 +1 @@
-consule.log('hello from github')
+console.log('hello from github')
```
Comment (circuitstoaim) [inline main.js]: why ? sounds nice

Comment (circuitstoaim) [reply main.js]: @kenzoTange kenzo whyyyyyyyyyyyyy

**Comment by circuitstoaim** [comment]
hello

**Comment by kenzoTange** [comment]
hello alreza

File: main.js (hunk 1 above)
Comment (kenzoTange) [inline main.js]: console is right

Comment (kenzoTange) [reply main.js]: because

**Pull request merged**